app/services/onetime/revert_to_product_level_refund_policies.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+# Used to process select sellers manually
+# For the rest, we'll use `seller_refund_policy_disabled_for_all` feature flag to override User#refund_policy_enabled flag
+#
+
+class Onetime::RevertToProductLevelRefundPolicies < Onetime::Base
+  LAST_PROCESSED_ID_KEY = :last_processed_seller_for_revert_to_product_level_refund_policy_id
+
+  attr_reader :seller_ids, :invalid_seller_ids
+
+  def self.reset_last_processed_id
+    $redis.del(LAST_PROCESSED_ID_KEY)
+  end
+
+  def initialize(seller_ids: [])
+    raise ArgumentError, "Seller ids not found" if seller_ids.blank?
+
+    @seller_ids = seller_ids
+    @invalid_seller_ids = []
+    @last_processed_index = ($redis.get(LAST_PROCESSED_ID_KEY) || -1).to_i
+  end
+
+  def process
+    ReplicaLagWatcher.watch()
+    seller_ids.each_with_index do |seller_id, index|
+      if index <= @last_processed_index
+        Rails.logger.info "Seller: #{seller_id} (#{index + 1}/#{seller_ids.size}): skipped (already processed in previous run)"
+        next
+      end
+
+      message_prefix = "Seller: #{seller_id} (#{index + 1}/#{seller_ids.size})"
+      seller = User.find(seller_id)
+      if !seller.account_active?
+        Rails.logger.info "#{message_prefix}: skipped (not active)"
+        next
+      end
+
+      if seller.refund_policy_enabled?
+        seller.with_lock do
+          seller.update!(refund_policy_enabled: false)
```

Comment:
This was set via similar scripts in Jan.

https://github.com/antiwork/gumroad/blob/932796780661406e266e9ff4ba18bcbfeafe423b/app/services/onetime/enable_refund_policy_for_sellers_with_refund_policies.rb
https://github.com/antiwork/gumroad/blob/932796780661406e266e9ff4ba18bcbfeafe423b/app/services/onetime/enable_refund_policy_for_sellers_without_refund_policies.rb